examples/fdc3-chart-and-grid/fdc3-chart-and-grid.sikuli/grid-and-chat-display-automated-test.py

```diff
@@ -0,0 +1,13 @@
+if exists("./screenshots/modules"):
```

Comment:
Quite unfortunately this is relative to the working directory, and not the script. We can live with this, or solve it later, but it's worth noting in the readme